setup.sh

```diff
@@ -91,6 +93,8 @@ CHANNEL="${ARR_CHANNEL[0]}"
 [[ -z $ARCH ]] && ARCH=x64
 [[ -z $CACHE_PATH ]] && CACHE_PATH="$RUNNER_TEMP/flutter/:channel:-:version:-:arch:"
 [[ -z $CACHE_KEY ]] && CACHE_KEY="flutter-:os:-:channel:-:version:-:arch:-:hash:"
```

Comment:
Let's remove `-:hash:` from here? It makes self-hosted runners to re-download flutter from cache every time.

Reply:
@vladimir-goldobin I'm not clear as to why `:hash:` is a problem here? As I understand it, that's the hash from the manifest (e.g. https://storage.googleapis.com/flutter_infra_release/releases/releases_macos.json) and the hash of the release, independent of the runner.

Reply:
Yes, I think `:hash:` shouldn't be a problem here.